source/MaterialXGenHw/HwShaderGenerator.cpp

```diff
@@ -3,196 +3,18 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <MaterialXGenShader/HwShaderGenerator.h>
+#include <MaterialXGenHw/HwShaderGenerator.h>
```

Comment:
As above, let's organize these into a block for `MaterialXGenHw`, followed by a block for `MaterialXGenShader`.